JournalCoreData/View Controllers/EntryDetailViewController.swift

```diff
@@ -68,7 +72,9 @@ class EntryDetailViewController: UIViewController {
     
     var entry: Entry? {
         didSet {
-            updateViews()
+            if isViewLoaded {
```

Comment:
I would recommend that you move the check for isViewLoaded into your updateViews() method, so you don't mess it up in other places.

```swift
    private func updateViews() {
        guard let entry = entry,
            isViewLoaded else {
                title = "Create Entry"
                return
        }
        
        let attributes = [NSAttributedString.Key.foregroundColor : UIColor.white]
        moodSegmentedControl.setTitleTextAttributes(attributes, for: .selected)
```